config/v3_2_experimental/translate/translate.go

```diff
@@ -38,10 +38,39 @@ func translateStorage(old old_types.Storage) (ret types.Storage) {
 	return
 }
 
+func translatePasswdUser(old old_types.PasswdUser) (ret types.PasswdUser) {
```

Comment:
The default value for `shouldExist` will be `nil`, right?  Do we need this translator?

Reply:
Yes, we do.  Automatic translation only handles identical structs.